client/main.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+import React from 'react';
+import { render } from 'react-dom';
+import Http from 'axios/dist/axios';
```

Comment:
:bowtie: I understand where did this come from but it is simply wrong...
According to [source](https://github.com/mzabriskie/axios/blob/master/dist/axios.js#L94) axios exports prebaked axios instance that has `#create` factory attached alongside with `Axios` class exposed same way.
Pascal casing is used to indicate that value is a class while in reality you imported instance of a class 😵 
It would be really crazy to see something like:
```js
let client = new Http.Axios(options);
```
Long story short it should be `http`.

Additionally don't go into webpack's way. Bundling already bundled code disables any possible optimisation that webpack might apply to it. Webpack should warn you about that one. 🤔  Correct way is to include source variant ie:
```js
import http from 'axios'
```